README.rst

```diff
@@ -0,0 +1,8 @@
+``pyramid_favicon``
+===================
+
+``pyramid_favicon``	is an add-on for handling favicons in your Pyramid project. Normally,
```

Comment:
There is too much spaces (or is there a tab?) between ``pyramid_favicon`` and ``is``